igor-web/src/test/groovy/com/netflix/spinnaker/igor/concourse/service/ConcourseServiceSpec.groovy

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright 2016 Schibsted ASA.
```

Comment:
While I'd love for my organisation to take credit for your test, you might want to update this 😉

Reply:
done